model/registry/llm.yaml

```diff
@@ -0,0 +1,189 @@
+groups:
+  - id: registry.llm
+    prefix: llm
+    type: attribute_group
+    brief: >
+      This document defines the attributes used to describe telemetry in the context of LLM (Large Language Models) requests and responses.
+    attributes:
+      - id: request.vendor
+        type: string
+        brief: The name of the LLM foundation model vendor, if applicable.
+        examples: 'openai'
+        tag: llm-generic-request
+      - id: request.model
+        type: string
+        brief: The name of the LLM a request is being made to.
+        examples: 'gpt-4'
+        tag: llm-generic-request
+      - id: request.max_tokens
+        type: int
+        brief: The maximum number of tokens the LLM generates for a request.
+        examples: [100]
+        tag: llm-generic-request
```

Comment:
nit: instead of tags you can define multiple groups like `llm.request`, `llm.response`, etc to make table generation easier